### ISSUE TITLE
Up v0.2.2 on node v0.8.1

### ISSUE THREAD

**Issue by yields** [LearnBoost/up#40]
Is `up` ready for production ?

There is a weird problem with it when used with express.
In a nutshell when a route is created and then deleted sometimes it will
work and sometimes it wouldn't work, you load the page it works, you reload it doesn't
and on and on...

I thought it was chrome cache, i deleted all cache `since the beginning of time` and still the same thing.
when i kill `up` and attempt to run it again `up -w -p 3000 app.js` i would get `address in use error`,
for `up` to work again i need to `killall node` and then `up` runs.

Another problem is with `up` code reload, i have 20 node processes for a single app, is this normal ?

**Comment by bjconlan** [comment]
I'm on 8.1 here (on ubuntu 12.04/i5) and don't have such problems regarding the number of node processes (correctly identifies 4 by default). And no, I don't believe that 20 spawned node processes is normal.

Perhaps check the default number of spawned instances using `require('os').cpus().length;`.

I can't answer the 'production ready' question, but we use it on our staging server at the moment and if it holds up over the next month I don't see a reason why we wouldn't also use it for our production system.

**Comment by yields** [comment]
Yeah i have a  staging server as well ( same as yours! :) ) and don't have any problems with `up`,
but those problems are on my local machine `mac os 10.8`.

i think @guille have a mac ?
